auto_configurator/conf/config.yaml

```diff
@@ -18,7 +18,7 @@ launcher_scripts_path: ${auto_configurator_path}/../launcher_scripts
 base_results_dir: ${auto_configurator_path}/results
 data_dir: ${launcher_scripts_path}/data
 
-training_container: nvcr.io/ea-bignlp/ga-participants/nemofw-training:23.11
+training_container: nvcr.io/ea-bignlp/ga-participants/nemofw-training:24.01
```

Comment:
can you update all links to `nvcr.io/nvidia/nemo:24.01.framework` ?

Reply:
updated